source/help/apps/desktop-changelog.md

```diff
@@ -0,0 +1,333 @@
+ - Added validation for name and URL when adding a new team on the Settings page
+ - The app now tries to reconnect periodically if a page fails to load
+ - Only one instance of the desktop application will now load at a time
+
+#### OS X
+ - Current team and channel name shown in window title bar
+ - Team tab is bolded for unread messages and has a red dot with a count of unread mentions
+ - Added an option to configure whether a red badge is shown on taskbar icon for unread messages
+ - Added new shortcuts:
+     - `CMD + S`: sets focus on the Mattermost search box
+     - `CMD + SHIFT + S`: sets focus on the Mattermost search box and adds `in:[channel_url]` to search in current channel
+     - `CMD + [`: go to previous page in history
+     - `CMD + ]`: go to next page in history
+ - Upgraded the Settings page user interface
+ - The app now tries to reconnect periodically if a page fails to load
+ - Added validation for name and URL when adding a new team on the Settings page
+
+#### Linux (Beta)
+ - Current team and channel name shown in window title bar
+ - Team tab is bolded for unread messages and has a red dot with a count of unread mentions
+ - Added an option to flash taskbar icon when a new message is received
+ - Added a red badge to count mentions on the taskbar icon (for Unity)
+ - Added a script to create `Mattermost.desktop` desktop entry to help [integrate the application into a desktop environment](https://wiki.archlinux.org/index.php/Desktop_entries) more easily
+ - Added new shortcuts:
+     - `CTRL + S`: sets focus on the Mattermost search box
+     - `CTRL + SHIFT + S`: sets focus on the Mattermost search box and adds `in:[channel_url]` to search in current channel
+     - `ALT + Left Arrow`: go to previous page in history
+     - `ALT + Right Arrow`: go to next page in history
+ - Upgraded the Settings page user interface
+ - Added access to the settings menu from the system tray icon
+ - The app now tries to reconnect periodically if a page fails to load
+ - Added validation for name and URL when adding a new team on the Settings page
+ - Only one instance of the desktop application will now load at a time
+
+### Bug Fixes
+
+#### Windows
+ - Cut, copy and paste are shown in the user interface only when the commands are available
+ - Copying link addresses now work properly
+ - Saving images by right-clicking the image preview now works
+ - Refreshing the app page no longer takes you to the team selection page, but keeps you on the current channel
+ - Removed misleading shortcuts from the system tray menu
+ - Removed unclear desktop notifications when the application page fails to load
+ - Fixed the Mattermost icon for desktop notifications in Windows 10
+ - Fixed an issue where the maximized state of the app window was lost in some cases
+ - Fixed an issue where shortcuts didn't work when switching applications or tabs in some cases
+ - Fixed a pixelated application icon in top left of the window
```

Comment:
should it be fixed "an issue where application icon _at_ the top left of the window was pixelated"?